src/connectors/aws_inventory.py

```diff
@@ -327,6 +373,38 @@ def elb_dispatch(landing_table, aws_access_key='', aws_secret_key='', accounts=N
     return results
 
 
+def ingest_iam(landing_table, aws_access_key=None, aws_secret_key=None, session=None, account=None):
+    users = get_iam_users(
+        aws_access_key=aws_access_key,
+        aws_secret_key=aws_secret_key,
+        session=session,
+        account=account
+    )
+
+    monitor_time = datetime.utcnow().isoformat()
+
+    print(db.derive_insert_select(LANDING_TABLES_COLUMNS))
+    print(db.derive_insert_columns(LANDING_TABLES_COLUMNS))
+    db.insert(
+        landing_table,
+        values=[(
+            row,
+            monitor_time,
+            row['Path'],
+            row['UserName'],
+            row['UserId'],
+            row.get('Arn'),
+            row['CreateDate'],
+            row.get('PasswordLastUsed'))
+            for row in users
+        ],
+        select=db.derive_insert_select(LANDING_TABLES_COLUMNS),
```

Comment:
`LANDING_TABLE_COLUMNS` references all the landing tables for all types of ingestion. I think you want `LANDING_TABLES_COLUMNS['IAM']`